### PR TITLE
README.md: lint, make slightly less horrible

### PR DESCRIPTION
**Pull request by morancj**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,134 +37,139 @@ For more information, please see the [build container wiki](https://github.com/l
...
 
 ![Making a change and creating a pull request.](/assets/images/help/CreatingAPullRequest.gif)
 
-
 #### Overriding the need for a reviewer on your pull request
 
 If you are certain that you need to merge the changes you have made and you are an Administrator of the repository you do the following to merge the PR.
 
-**Warning:** If you are an administrator and are using the above method of mergin your pr. Then please make sure that at least the tests __have passed__!
+**Warning:** If you are an administrator and are using the above method of mergin your PR, please ensure that the tests __have passed__ before force merging!
```
Comment (pcolmer) [inline README.md]: "mergin" is missing a trailing 'g'.

I know it was always missing but if you are going to edit the line ...

File: README.md [hunk 2]
```diff
@@ -45,134 +37,139 @@ For more information, please see the [build container wiki](https://github.com/l
...
-...
-{% include sections/keynotes.html keynotes=site.data.yvr18-keynotes %}
-...
-```
+    ...
+    {% include sections/keynotes.html keynotes=site.data.yvr18-keynotes %}
+    ...
 
-E.g 
+E.g
```
Comment (pcolmer) [inline README.md]: There should be a trailing full-stop.

Comment (morancj) [reply README.md]: Replaced

**Comment by morancj** [comment]
@kylekirkby @pcolmer The VS Code extension [Spell Right](https://marketplace.visualstudio.com/items?itemName=ban.spellright) works well with Markdown. :smiley:

**Pull request merged**